dynamo/preprocessing/utils.py

```diff
@@ -707,12 +707,12 @@ def normalize_mat_monocle(
     if pseudo_expr is None:
         pseudo_expr = 1
 
-    if issparse(mat):
-        mat.data = norm_method(mat.data + pseudo_expr) if norm_method is not None else mat.data
-        if norm_method is not None and norm_method.__name__ == "Freeman_Tukey":
-            mat.data -= 1
-    else:
-        mat = norm_method(mat + pseudo_expr) if norm_method is not None else mat
+    # if issparse(mat):
+    #     mat.data = norm_method(mat.data + pseudo_expr) if norm_method is not None else mat.data
+    #     if norm_method is not None and norm_method.__name__ == "Freeman_Tukey":
+    #         mat.data -= 1
+    # else:
+    #     mat = norm_method(mat + pseudo_expr) if norm_method is not None else mat
```

Comment:
there are a few things here: 
1. please clean up the function so there is no the norm_method argument. 
2. please write a new function called transform_normarlized_data and let it support log1p and other transformation

Reply:
After careful consideration, we have decided to keep the normalize_mat_monocle function since it is called from many different places. However, we will set the skip_log parameter to True so that we can skip the log transformation step, as the input requires.

Reply:
Done